packages/fxa-content-server/app/scripts/views/password_strength/password_with_strength_balloon.js

```diff
@@ -24,8 +24,7 @@ const PasswordWithStrengthBalloonView = FormView.extend({
   events: {
     change: 'updateModelAfterDelay',
     focus: 'focusHandler',
-    keypress: 'updateModelAfterDelay',
-    blur: 'blurHandler',
+    keydown: 'updateModelAfterDelay',
```

Comment:
Doesn't a keydown also trigger a change event, which is handled by `updateModelAfterDelay` a couple lines up?

Reply:
Ooo @chenba good call out - it doesn't look like we need `change` here at all. `keypress` also fired `updateModelAfterDelay` but this may have been missed due to the debounce. I took out `change` and readded `blur`, and passed the `event.type` to `updateModel` instead of using jQ to check if the el was out of focus. I also made a comment clearer, added a test for `inputFocused` on the model, and updated `shouldHide` to `hideOrShow`.

I checked all the edgecases and tried to break the thing and it's looking good to me. Clicking on password reveal all works as expected too.